task-sdk/tests/conftest.py

```diff
@@ -261,155 +259,3 @@ def _make_context_dict(
         return context.model_dump(exclude_unset=True, mode="json")
 
     return _make_context_dict
-
-
-@pytest.fixture
```

Comment:
All of these isn't required since it is moved to `devel-common/src/tests_common/pytest_plugin.py`